frontend/src/components/Landing/Faq.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { HashLink } from 'react-router-hash-link';
+
+import styles from './faq.cssmodule.scss';
+
+const Faq = () => (
+  <div className={styles.container}>
+    <div className={styles.description}>
+      <div>Vous avez une question ?</div>
+      <div>
+        Consultez la Foire aux questions
```

Comment:
`Foire` here and `foire` in the button, maybe we could use same case in both place.